docs/containerd_config.md

```diff
@@ -1,4 +1,4 @@
-# containerd configuration
+# Containerd configuration
```

Comment:
official name is with lower case c `https://containerd.io/docs/`

Reply:
changed